nixos/modules/installer/tools/nixos-rebuild.sh

```diff
@@ -27,6 +27,7 @@ profile=/nix/var/nix/profiles/system
 buildHost=
 targetHost=
 maybeSudo=()
+NIX="nix --option experimental-features nix-command"
```

Comment:
We can't do this, because passing `--experimental-features` overrides the setting in nix.conf. So it might *disable* other features.

Reply:
That is really unfortunate, it means we cannot dogfood nix commands.